Forge/src/main/resources/META-INF/mods.toml

```
@@ -19,7 +19,7 @@ issueTrackerURL="https://github.com/Tschipp/CarryOn/issues"
     # Does this dependency have to exist - if not, ordering below must be specified
     mandatory=true #mandatory
     # The version range of the dependency
-    versionRange="[46,47.1.3]" #mandatory
+    versionRange="[46,47.1.47]" #mandatory
```

This will break on the new Forge recommended build 47.2.0. Instead, it should be something like this:
```suggestion
    versionRange="[46.0.0,47.1.3],[47.1.43,)" #mandatory
```